script/build.js

```diff
@@ -61,6 +62,38 @@ if (options.buildtype === undefined) {
   options.buildtype = 'development';
 }
 
+if (process.env.HEROKU_APP_NAME) {
+  const request = require('sync-request');
+  options.destination = path.resolve(__dirname, '../build/heroku');
+  try {
+    const pr = process.env.HEROKU_APP_NAME.match(/pr-(\d+)$/)[1];
+    const res = request(
+      'GET',
+      `https://api.github.com/repos/department-of-veterans-affairs/vets-website/pulls/${pr}`,
+      {
+        headers: {
+          'User-Agent': 'vets-website-builder'
+        },
+        json: true
+      }
+    );
+    const respObj = JSON.parse(res.getBody('utf8'));
+
```

Comment:
Ah, it's a shame to see another HTTP module in vets-website, but since the operation has to be done synchronously I don't think there's currently an easy way to do that installed.

Reply:
Yeah, I spent a bunch of time trying to avoid pulling in yet another library but didn't find any other options that would work.